smarts/env/gymnasium/wrappers/metrics.py

```diff
@@ -291,7 +291,7 @@ def score(self) -> Score:
         humanness = _humanness(costs=costs_tot, agents_tot=agents_tot)
         rules = _rules(costs=costs_tot, agents_tot=agents_tot)
         time = _time(counts=counts_tot)
-        overall = completion * (1 - time) * (1 - humanness) * (1 - rules)
+        overall = completion * (1 - time) * humanness * rules
```

Comment:
I thought this was the case already. It looks like I may have caused this when I moved the metrics.

Reply:
Corrected in PR #1848 .